app/views/nsm/steps/case_category/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<% title t('.page_title') %>
+<% decision_step_header %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary(@form_object) %>
+
+    <%= step_form @form_object do |f| %>
+      <%= f.govuk_radio_buttons_fieldset :plea_category, legend: -> { radio_legend(t('.heading')) } do %>
```

Comment:
I'm curious about this. In loads of places we're doing `legend: { text: t('.legend'), size: 'xl', tag: 'h1' }`. What's the problem with doing that, and can that problem be explained as a comment in the `radio_legend` method?

Reply:
tbh, I don't thnk there's a problem with this - I just didn't know the correct syntax and went down kind of a roundabout route that was based on an example in the documentation, will update this and remove the helper

it does seem like some of the other pages with radio button options might need updating though as they have this legend as the page heading instead (which doesn't follow GDS) - but that's a different story